indidriver.c

```diff
@@ -234,6 +234,10 @@ int IUUpdateSwitch(ISwitchVectorProperty *svp, ISState *states, char *names[], i
     ISwitch *sp;
     char sn[MAXINDINAME];
 
+    if (svp == 0) {
```

Comment:
"nullptr" instead? Else looks good obviously, but we'd better find the root cause for sure.

Reply:
nullprt is a C++11 construct. This is a pure C source file.

Reply:
Correct. assert should be used in such a case.

Reply:
@gubihu agreed, sorry.